app/com/yetu/controlcenter/utils/WSUtils.scala

```diff
@@ -0,0 +1,81 @@
+package com.yetu.controlcenter.utils
+
+import play.api.libs.ws.WSRequestHolder
+import play.api.mvc.Result
+import play.api.mvc.Results.Status
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object WSUtils {
+
+
+  /**
+   * * Creates and return Authorization Header to be used in WSRequestHolder
+   * @param accessToken
+   * @return
```

Comment:
I wonder if we can get rid of these 
```
+ * @param accessToken
+ * @return
```
lines? They kind of clutter the code and don't add much value.